portal/static/js/initialQueries.js

```diff
@@ -473,6 +473,12 @@
                 $("#aboutForm").fadeIn();
             }
         }
+        if ($("#next").hasClass("accept-on-next")) { //related to MUSIC - Accept on Next - special case
```

Comment:
I'd rather avoid business rules in the code - is this specific to MUSIC, or more general if a config turns on the `accept-on-next` feature?

Reply:
therefore, i'd be tempted to drop the "related to MUSIC" and make the comment more like what you just told me :)